File: query_feature_266.js
const Database = require('/Users/justinbundrick/Documents/ALAESolutions/RIMSS/RIMSS-autocoder/backend/node_modules/.pnpm/better-sqlite3@12.6.2/node_modules/better-sqlite3');
const db = new Database('./features.db', { readonly: true });

const feature = db.prepare('SELECT * FROM features WHERE id = ?').get(266);
console.log(JSON.stringify(feature, null, 2));

db.close();

File: query_feature_266.py
import sqlite3
import json

conn = sqlite3.connect('features.db')
cursor = conn.cursor()
cursor.execute('SELECT * FROM features WHERE id = 266')
row = cursor.fetchone()

if row:
    columns = [description[0] for description in cursor.description]
    feature = dict(zip(columns, row))
    print(json.dumps(feature, indent=2))
else:
    print("Feature not found")

conn.close()
